.build/CBT/build.props

```diff
@@ -81,21 +81,31 @@
 
   </PropertyGroup>
 
+  <ItemGroup>
+    <CBTParseError Condition=" '$(EnlistmentRoot)' == '' " Include="The 'EnlistmentRoot' property must be set.  Please ensure it is declared in a properties file before CBT Core is imported.">
+      <Code>CBT1000</Code>
+    </CBTParseError>
+    <CBTParseError Condition=" '$(CBTModulesRestored)' == 'false' " Include="Modules were not restored and the build cannot continue.  Refer to other errors for more information.">
+      <Code>CBT1001</Code>
+    </CBTParseError>
+    <CBTParseError Condition=" '$(CBTModulePackageConfigPath)' == '' " Include="The 'CBTModulePackageConfigPath' is not set or no package configuration file was found in the expected location.  You must have a packages.config or project.json file that specifies what modules to use.">
```

Comment:
Can you fix this message while you're in here: "The 'CBTModulePackageConfigPath' **property** is not set..."